prow/cmd/deck/static/script.js

```diff
@@ -353,7 +353,11 @@ function equalSelected(sel, t) {
 }
 
 function groupKey(build) {
-    return build.repo + " " + build.number + " " + build.refs;
+    return build.spec.refs.org + "/" +
```

Comment:
This won't do the right grouping for batch jobs. Try this:

    const refs = build.spec.refs;
    const pulls = refs.pulls.map(r => r.number + ':' + r.sha).join(',');
    return `${refs.repo} ${refs.base_ref}:${refs.base_sha},${pulls}`;